openwisp_firmware_upgrader/tests/test_models.py

```
@@ -699,3 +702,244 @@ def test_delete_firmware_files_on_organization_delete(self):
         org.delete()
         # Check that the file was deleted
         self.assertEqual(file_storage_backend.exists(file_name), False)
+
+    @mock.patch(_mock_updrade, return_value=True)
+    @mock.patch(_mock_connect, return_value=True)
+    def test_batch_upgrade_with_group_filtering(self, *args):
```

Ruff (0.14.14)

Unused method argument: `args`

(ARG002)